core/src/main/java/feast/core/model/Project.java

```diff
@@ -34,6 +34,7 @@
 @Entity
 @Table(name = "projects")
 public class Project {
+  public static final String DEFAULT_NAME = "default";
```

Comment:
Also, is the default project not configurable?

Reply:
Yes its not configurable.